### PR TITLE
app-laptop/tiny-dfr: new package, add 0.2.0

### PR DESCRIPTION
**Pull request by WhatAmISupposedToPutHere**
This is a daemon that drives the "Touch Bar" (a touch screen replacing the F{n} keys) on some arm macbooks.

### CODE REVIEW

**Comment by thesamesam** [comment]
cc @leio @chadmed

**Comment by chadmed** [comment]
Straight from the overlay so LGTM. I believe @leio had some thoughts about the whole systemd/openrc thing. Maybe we should just install both the unit file and init script unconditionally?

I would probably add the Asahi Project to `metadata.xml` too but that's neither here nor there.

File: app-laptop/tiny-dfr/tiny-dfr-0.2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+}
+
+src_install() {
+	cargo_src_install
+
+	insinto /usr/share/tiny-dfr
+	doins share/tiny-dfr/*
+
+	udev_dorules etc/udev/rules.d/*
+	use systemd && systemd_dounit etc/systemd/system/tiny-dfr.service
```
Comment (thesamesam) [inline app-laptop/tiny-dfr/tiny-dfr-0.2.0.ebuild:246]: Per https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301, please drop the USE and just install them unconditionally.

**Comment by thesamesam** [comment]
Many thanks! Consider joining us on IRC at #gentoo-arm on libera and also #asahi-alt on OFTC if not there already too (you may well be at the latter at least, I've not got to know everyone yet).

**Pull request closed without merging**